_config.yml

```diff
@@ -9,6 +9,7 @@ url: "http://developer.fedoraproject.org" # the base hostname & protocol for you
 twitter_username: jekyllrb
 github_username:  jekyll
 encoding: utf-8
+fedora_release: 34
```

Comment:
Why is it being declared here twice? Let's prevent duplication.

The latter declaration could work.

Reply:
it was already on the bottom but i don't have jekyll knowledge to reach that variable so i redeclared it as a 'site' global variable.

figured id leave the old config as it was so i don'T fuck up things

Reply:
it wouldnt generate a version without adding it in the general/default site variable